.env.local.example

```diff
@@ -14,6 +14,10 @@ REDIS_URL=
 REDIS_PASSWORD=
 REDIS_EMAIL_TO_ID_SECRET=
 
+# If you'd like to use Strapi as your CMS
```

Comment:
This should be moved to the end to respect the order added 👍

Reply:
All good now!